barcode/.gitignore

```diff
@@ -0,0 +1 @@
+version.py
```

Comment:
This is already ignored by [`.gitignore`](https://github.com/WhyNotHugo/python-barcode/blob/4c0210f22562539b9948eba09b70094ec4f4a036/.gitignore#L17), so this is redundant.